test/powershell/Modules/Microsoft.PowerShell.Management/Clear-Content.Tests.ps1

```diff
@@ -75,35 +80,54 @@ Describe "Clear-Content cmdlet tests" -Tags "CI" {
       $cci.SupportsShouldProcess | Should -BeTrue
     }
 
-    It "Alternate streams should be cleared with clear-content" -Skip:(!$IsWindows) {
-      # make sure that the content is correct
-      # this is here rather than BeforeAll because only windows can write to an alternate stream
-      Set-Content -Path "TestDrive:/$file3" -Stream $streamName -Value $streamContent
-      Get-Content -Path "TestDrive:/$file3" | Should -BeExactly $content2
-      Get-Content -Path "TestDrive:/$file3" -Stream $streamName | Should -BeExactly $streamContent
-      Clear-Content -Path "TestDrive:/$file3" -Stream $streamName
-      Get-Content -Path "TestDrive:/$file3" | Should -BeExactly $content2
-      Get-Content -Path "TestDrive:/$file3" -Stream $streamName | Should -BeNullOrEmpty
-    }
-
-    It "the '-Stream' dynamic parameter is visible to get-command in the filesystem" -Skip:(!$IsWindows) {
-      try {
-        Push-Location -Path TestDrive:
-        (Get-Command Clear-Content -Stream foo).parameters.keys -eq "stream" | Should -Be "stream"
+    Context "Clear-Content should work with alternate data streams on Windows" {
+      BeforeAll {
+        if (!$IsWindows)
+        {
+          return
+        }
+        Set-Content -Path "TestDrive:/$file3" -Stream $streamName -Value $streamContent
+        Set-Content -Path "TestDrive:/$dirName" -Stream $streamName -Value $streamContent
```

Comment:
It seems we can remove the two lines.

Reply:
We can get rid of this entire BeforeAll block, if we don't need those two lines.